src/SDKs/SqlManagement/Management.Sql/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -10,4 +10,7 @@
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />
```

Comment:
@jaczhan any reason you need this nuget in your SQL package?
The requirement was only for recording new test, so your other change is all you need.
You don't want additional dependency on your SQL SDK package.
Please revert "this" particular change.
Please keep your other change that you made in the test project (that is what I had asked for)